nekoyume/Assets/_Scripts/Blockchain/StateGetter.cs

```diff
@@ -0,0 +1,217 @@
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Security.Cryptography;
+using Bencodex.Types;
+using Libplanet.Common;
+using Libplanet.Crypto;
+using Libplanet.Types.Assets;
+using Nekoyume.Action;
+using Nekoyume.Exceptions;
+using Nekoyume.Model.Arena;
+using Nekoyume.Model.Item;
+using Nekoyume.Model.Stake;
+using Nekoyume.Model.State;
+using Serilog;
+
+namespace Nekoyume.Blockchain
+{
+    public static partial class StateGetter
+    {
+        public static IValue GetState(Address address, HashDigest<SHA256> hash) =>
+            Game.Game.instance.Agent.GetState(address, hash);
+
+        public static bool TryGetState(Address address, HashDigest<SHA256> hash, out IValue value)
+        {
+            value = Game.Game.instance.Agent.GetState(address, hash);
+            return value is not Null;
+        }
```

Comment:
This `TryGetXyz()` method need move to `StateGetter.TryGet` I guess.

Reply:
https://github.com/planetarium/NineChronicles/pull/3138/commits/9d2b02ca1d34c01b66558c5be74a21b5195c4e42 amended!